### ISSUE TITLE
Suggestion: Remember RTL with property

### ISSUE THREAD

**Issue by JonDror** [sawhney17/logseq-rtl-mode-plugin#1]
Hi,

**THANK YOU** for this plugin!

Suggestions to add: Remember the orientation of a page.

- This can be done by adding a property to pages that have RTL on. like: `direction:: rtl`
- Then, English pages can stay in LTR, while Arabic/Hebrew pages only need to be toggled once (or manually add the property) and the RTL orientation will be "remembered" each time the page is viewed.

**Comment by iahmed0199** [comment]
Yeah, I'm pretty sure about that feature it would be beneficial for my use too...
